### DISCUSSION TITLE
Higher CPU count virtual environments planned?

### DISCUSSION THREAD

**Discussion by hakusaro** [community/community#26570] [Actions]
<p>Hiya! I couldn’t find any information laying around in the existing GitHub actions documentation about higher CPU instances being available. I was wondering if anyone knew of plans to offer build agents with higher CPUs in the future? Sadly I have a test suite that takes 35 minutes to run in GitHub Actions because it’s highly parallel, and 2 CPUs doesn’t scale well.</p>
<p>This is a paid org plan, so I’d be more than happy to consider higher cost instances if available.</p>

**Comment by ethomson** [comment, marked as answer]
<p>Yes, we’ll have more compute options available in the future.  Our first offering here will be the ability to provide self-hosted runners, so you can bring your own compute needs to GitHub Actions.  We’ll explore additional offerings after that.</p>

**Comment by savv** [comment]
<p>Is there any update on this? For building android apps, it would be great to have 4 or 8 cores so that it can go faster.</p>

**Comment by ohsayan** [comment]
<p>Would love to have upgraded plans for more cores too. Any updates?</p>

**Comment by petrvecera** [comment]
Looks like this was just released https://github.blog/changelog/2022-09-01-github-actions-larger-runners-are-now-in-public-beta/
